tests/wpunit/PostObjectConnectionQueriesTest.php

```diff
@@ -755,5 +755,89 @@ public function testPostExcerptsAreDifferent() {
 		$this->assertNotEquals( $edges[0]['node']['content'], $edges[1]['node']['content'] );
 
 	}
-	
+
+	private function formatNumber($num) {
+		return sprintf('%08d', $num);
+	}
+
+	public function assertMetaQuery( $meta_fields ) {
+
+		add_filter( 'graphql_map_input_fields_to_wp_query', function( $query_args ) use ( $meta_fields ) {
+			return array_merge( $query_args, $meta_fields );
+		}, 10, 1 );
+
+		// Must use dummy where args here to force
+		// graphql_map_input_fields_to_wp_query to be executes
+		$query = "
+		query getPosts(\$cursor: String) {
+			posts(after: \$cursor, first: 5, where: {author: {$this->admin}}) {
```

Comment:
@epeli I don't see where the `order` is being set here in the query? What am I missing? 🤔

Reply:
Its added by the `graphql_map_input_fields_to_wp_query` filter

Reply:
ah, I think I get it. You're testing that the capability works even if the query itself doesn't specify order, because the server may want to set a default order? eh?

Reply:
I guess I was assuming the test would have order set in the query like:

```
posts(after: \$cursor, first: 5, where: {author: {$this->admin}, orderBy: { "someMetaKey" } }) {
```

or something to that tune, but I think I'm on the same page now

Reply:
Yeah, in real world code would do it like that. Just taking a shortcut in the tests.